crates/summit/static/style.css

```
@@ -0,0 +1,374 @@
+html {
+    height: 100%;
+}
+
+body {
+    background-color: #170c1c;
+    font-family: sans-serif;
+    height: 100%;
```

Seems weird to have `body`s children be bigger than their parent. I'm guessing that this is so the background color extends to the bottom of the viewport if the body contents don't fill the viewport? I'd solve this using

```suggestion
    min-height: 100%;
```

---

Ah yeah good catch